codewof/templates/users/dashboard.html

```diff
@@ -30,11 +30,23 @@ <h2>Your questions for today</h2>
             <strong>Sorry!</strong> We currently don't have questions for you to do today.
           </div>
         {% endif %}
+        <p>You've solved {{num_questions_answered}} question{{ num_questions_answered|pluralize }} in the last month!</p>
+<!--        {% include "users/activity_graph.html" %}-->
       </div>
 
       <div class="col-12 col-md-4 mb-3">
-        <h2>Achievements</h2>
-        <p><em>Coming soon!</em></p>
+        <h2>Points earned: {{user.profile.points}}</h2>
+        <h2><a id="achievements-link" href="{% url 'users:achievements' %}">Achievements</a></h2>
```

Comment:
It's hard to tell this is a link when none of the other identical text is. Maybe have `Achievements: X` like the line above and then a small `view all` button below

Or alternatively make the achievement icons themselves the link instead

Reply:
(I assume there's a max number of achievements displayed at once on that page?)

Reply:
Decided to implement the latter.